src/models/dataTypes.ts

```diff
@@ -92,7 +92,7 @@ export interface ICreateTaskBody<T> {
 export interface IUpdateTaskBody<T> {
   description?: string;
   parameters?: T;
-  status: OperationStatus;
+  status?: OperationStatus;
```

Comment:
As it is mandatory value in job-manager repo - please add link to job manager fix first
![image](https://github.com/MapColonies/mc-priority-queue/assets/18442671/4dc3a12a-6fde-4187-96fd-3279eab40e71)

Reply:
Done. Approve the PR there and then approve here :)